projects/ngrx.io/content/guide/store/action-groups.md

```diff
@@ -0,0 +1,61 @@
+# Action Groups
+
+Action groups is a feature that allows you to group actions together. While writing actions, the actions in most of the cases looks like below.
+
+```typescript
+import { createAction, props } from '@ngrx/store';
+
+export const opened = createAction('[Products Page] Opened');
+
+// defining an action with payload using the `props` function
+export const paginationChanged = createAction(
+  '[Products Page] Pagination Changed',
+  props<{ page: number; offset: number }>()
+);
+
+// defining an action with payload using the props factory
+export const queryChanged = createAction(
+  '[Product Page] Query Changed',
+  (query: Error) => ({ query })
+);
+```
+
+With the help of this new `createActionGroup` api, we can avoid the above code and write the following code.
+
+```typescript
+
+import { createActionGroup } from '@ngrx/store';
+
+export const productPageActions = createActionGroup({
+  source: '[Product Page]',
+  events: {
+    'Pagination Changed': props<{ page: number; offset: number }>(),
+    'Query Changed': (query: Error) => ({ query }),
+  }
+});
+
+```
+
+After moving to the new `createActionGroup` api, we can use the following code to import and dispatch the action.
+
+```typescript
+
+import { productPageActions } from './product.actions';
+
+productPageActions.paginationChanged({ page: 1, offset: 10 });
+productPageActions.queryChanged(new Error('Query is invalid'));
```

Comment:
Here we can also add an action type for each generated action:

```suggestion
import { ProductsPageActions } from './products-page.actions';

// action type: [Products Page] Opened
ProductsPageActions.opened();

// action type: [Products Page] Pagination Changed
ProductsPageActions.paginationChanged({ page: 10, offset: 100 });

// action type: [Products Page] Query Changed
ProductsPageActions.queryChanged('ngrx');
```

Reply:
Thanks 😊 done with the changes.